helm/pinot/templates/server/statefulset.yaml

```diff
@@ -88,15 +88,23 @@ spec:
           initialDelaySeconds: {{ .Values.probes.initialDelaySeconds }}
           periodSeconds: {{ .Values.probes.periodSeconds }}
           httpGet:
+            {{- if .Values.server.probes.livenessEndpoint }}
```

Comment:
two suggestions:
1. change the property to `.Values.server.livenessProbe.endpoint`
2. change line 86 to {{- if or .Values.server.probes.livenessEnabled .Values.server.livenessProbe.endpoint }}

Reply:
+1, let's make sure we have the properties following the conventions.

Reply:
Can you tell me why changing line 86 to if `Values.server.probes.livenessEnabled` OR `.Values.server.livenessProbe.endpoint` ?

Shouldn't if `Values.server.probes.livenessEnabled` be enough?